packages-next/keystone/src/scripts/build/build.ts

```diff
@@ -16,11 +16,15 @@ export async function build({ dotKeystonePath, projectAdminPath }: StaticPaths)
   console.log('✨ Generating graphQL schema');
   await saveSchemaAndTypes(graphQLSchema, keystone, dotKeystonePath);
 
-  console.log('✨ Generating Admin UI code');
-  await generateAdminUI(config, graphQLSchema, keystone, projectAdminPath);
+  if (config.ui?.isDisabled) {
+    console.log('✨ Skipping Admin UI build');
```

Comment:
This is problematic now that I think about it. The build also builds the API so we can't just skip it like this.